src/server/game/Entities/Object/Object.cpp

```diff
@@ -3152,7 +3152,7 @@ void WorldObject::GetNearPoint(WorldObject const* searcher, float &x, float &y,
 {
     GetNearPoint2D(searcher, x, y, distance2d, absAngle);
     z = GetPositionZ();
-    
+
```

Comment:
hello there?

Reply:
Fair point. Superfluous blank line, even if it was there before.

Reply:
i not even changed it, my visual that cleanup white space xD
i will remove when i merge it xD